hack/aks/Makefile

```diff
@@ -207,16 +236,18 @@ windows-cniv1-up: rg-up overlay-net-up ## Bring up a Windows CNIv1 cluster
 
 	$(AZCLI) aks nodepool add --resource-group $(GROUP) --cluster-name $(CLUSTER) \
 		--os-type Windows \
-		--os-sku $(WINDOWS_OS_SKU) \
+		--os-sku $(OS_SKU_WIN) \
 		--max-pods 250 \
 		--name npwin \
 		--node-count $(NODE_COUNT) \
-		-s $(WINDOWS_VM_SKU)
+		-s $(VM_SIZE_WIN)
 
 	@$(MAKE) set-kubeconf
 
 linux-cniv1-up: rg-up overlay-net-up
```

Comment:
also needs K8S_VER

Reply:
This is missing on all release train 🤦 . Will add it here and then make a PR to backport.

Reply:
PR created.